### PR TITLE
Document that the dashboard cache clear button will now do `wp-cli ca…

### PR DESCRIPTION
**Pull request by kibra**
…che flush`

### CODE REVIEW

**Comment by spheromak** [comment]
:+1:

File: source/docs/articles/wordpress/wordpress-faq.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Yes, we support Redis. Please see [Redis as a caching backend](/docs/articles/si
 
 ### Which caches do you clear?
 
-You can clear the caches from your site's Pantheon Dashboard. This will take care of clearing Varnish and Redis caches, if applicable. At this time, WordPress database caches are not cleared nor is selective Varnish cache clearing supported.
+You can clear the caches from your site's Pantheon Dashboard. This will take care of running `wp-cli cache flush` and clearing Varnish and Redis caches, if applicable. At this time, Varnish cache clearing supported.
```
Comment (ari-gold) [inline source/docs/articles/wordpress/wordpress-faq.md]: @kibra Typo? Is Varnish cache clearing supported?

Comment (bmackinney) [inline source/docs/articles/wordpress/wordpress-faq.md]: There's also a clear cache button in the WP admin dashboard. Does that do all of this too?

Comment (kibra) [inline source/docs/articles/wordpress/wordpress-faq.md]: @bmackinney The functionality of that button is not changing.

Comment (kibra) [inline source/docs/articles/wordpress/wordpress-faq.md]: @ari-gold Thanks for the catch. Fixing.

**Comment by bmackinney** [comment]
:+1:

**Pull request merged**